data/_ui/help.txt

```diff
@@ -25,6 +25,9 @@ help "basics 1"
 help "basics 2"
 	`For the main menu, press <Show main menu>. The control key bindings can be viewed and changed in the preferences.`
 
+help "basics 3"
+	`To see a help message again, press <Show help>. To see all help messages again, go to the main menu, go to preferences, go to settings, go to page two, then select "Reactivate first-time help."`
```

Comment:
Trying to reduce repetitiveness here.
Any reason why <Show help> is in brackets and "Reactivate first-time help" is in quotes?
```suggestion
	`To see a help message again, press <Show help>. To see all help messages again, go to the main menu, select Preferences, then go to Settings. Navigate to page two, then select "Reactivate first-time help."`
```

Reply:
It replaces Show help with your current key assigned to it (usually F1).